PANA - Action Management (#32744)
DataDog/documentation: content/en/product_analytics/guide/action_management.md

Your feedback is invaluable to Datadog during this preview stage. Share all of your feedback with the Product Analytics Product Manager, Milene Darnis, using [Datadog Public Slack](https://chat.datadoghq.com/).

## Setup 

### Step 1 - Install the browser extension 

Action Management requires [The Datadog test recorder Chrome extension][1]. If you are unable to add the extension through the Chrome web store, see the [manual instructions][2]. 

### Step 2 - Label your actions 

1. Go to the [Actions][3] page in the Datadog UI, and click on **Label New Action**. This takes you to the point-and-click interface where you can select your actions.

{{< img src="product_analytics/action_management/pana-labeled-new-action.png" alt="Actions page to start labelling" style="width:90%;">}}

2. Use the **Navigate Site** mode to browse to the location of the actions you want to label.

{{< img src="product_analytics/action_management/pana-point-click-interface.png" alt="The point and click interface used to locate and labed your action " style="width:90%;">}}

> 3. In **Label Actions** mode, click on a UI element to add and label it as an action. You can also click on the action you want to define to see more details about it.

Reviewer: ```suggestion
3. To add an action, first, switch to **Label Actions** mode.
```

{{< img src="product_analytics/action_management/pana-label-mode.png" alt="Click on and label your actions." style="width:90%;">}}

> 4. When you are ready, click on **New Labeled Action** to:

Reviewer: ```suggestion
4. Next, select the UI element that you would like to add as an action, click on **New Labeled Action**, and:
```

---
title: Action Management
private: true
---

> {{< callout url="#" btn_hidden="true" >}}

Reviewer: This callout, on the docs page, is titled "Join the Preview", but I don't see any instructions or button to actually join the preview

  Action Management is in preview.
{{< /callout >}} 

## Overview

> Action Management is a no-code way of labeling autocaptured actions in your website. Action Management helps to improve trust in your Product Analytics dataset and enhance efficiency in your analysis. This page guides you through getting started with using Action Management. 

Reviewer: (Just saw the above addressed in Known Limitations :) updated my suggestion)

As this feature is in preview, here are some important things to consider:

> - There are known limitations with the feature which Datadog is addressing. See the [limitations section](#known-limitations) for more information.

Reviewer: I don't think you need to mention the Known Limitations section here; it's a bit redundant

> - Your feedback is invaluable to Datadog during this preview stage. Share all of your feedback with the Product Analytics team using [Slack](https://chat.datadoghq.com/). If you do not have a Slack channel for sharing your feedback, send an message to Milene Darnis on the Datadog Public Slack.

Reviewer: This is a bit wordy and unclear as far as exactly what Slack channel they should use, but i'm not too familiar with the public Slack. I tightened up the language a bit.

## Setup 

### Step 1 - Install the browser extension 

> Labeling actions with Datadog Product Analytics requires this [Chrome extension][1]. See the [manual instructions if needed][2]. 

Reviewer: ```suggestion
Action Management requires [The Datadog test recorder Chrome extension][1]. If you are unable to add the extension through the Chrome web store, refer to the [manual instructions][2]. 
```

 

### Step 2 - Label your actions 

1. Go to the [Actions][3] page in the Datadog UI, and click on **Label New Action**. This takes you to the point-and-click interface where you can select your actions.

{{< img src="product_analytics/action_management/pana-labeled-new-action.png" alt="Actions page to start labelling" style="width:90%;">}}

> 2. Use the **Navigate Site** mode to browse through your website and identify the location of your actions. 

Reviewer: ```suggestion
2. Use the **Navigate Site** mode to browse to the location of the actions you want to label. 
```

{{< img src="product_analytics/action_management/pana-point-click-interface.png" alt="The point and click interface used to locate and labed your action " style="width:90%;">}}

> 3. Use the **Label Actions** mode, then click on each action to name your actions. 

Reviewer: ```suggestion
3. In **Label Actions** mode, click on a UI element to add and label it as an action. 
```

{{< img src="product_analytics/action_management/pana-label-mode.png" alt="Clik on and label your actions." style="width:90%;">}}

> 4. When in **Label Actions** mode, click on the action you want to define to see more details about it ( _Past 7 days volume_, _XPath_, _other Labeled Actions on the selected action_)

Reviewer: I think this should probably say "While in Label Actions mode, click on an action name to see more details about it." Without access to the UI, i'm not 100% sure that's accurate. 
Separately, I removed the names of the additional details, it doesn't read super cleanly and I think it's apparent enough in the UI what the user can do.

{{< img src="product_analytics/action_management/pana-new-label-action.png" alt="Define your selected actions." style="width:90%;">}}

5. When you are ready, click on **"New Labeled Action"** to:
- Limit the action on the current page or define an action for that selector on any page
- Limit to the target text on that page or define an action regardless of the target text (useful if your website is localized in several languages for instance)
- Give a name (required) and description (optional) to the action

You can also hold the Shift key to select other actions on the page and label several actions under the same name (is used as an “OR” in analysis). 

### Step 3 - Retrieve your Label your actions 

After you define an action, you can find them in the [list of labeled actions][4]. From there you can:
- Filter to only see your actions
- Edit actions

> - See this action in a funnel, retention chart or see related replays directly from there

Reviewer: ```suggestion
- View actions in a funnel, retention chart, related session replay
```

 {{< img src="product_analytics/action_management/pana-action-list.png" alt="List of your labeled actions." style="width:90%;">}}

## Known limitations 

> - Action Management only works for web pages at this time, mobile support is on the roadmap. 

Reviewer: ```suggestion
- Action Management only works for web pages at this time.
```

> - You cannot label Actions that are hidden behind a hover. If this limitation impacts your use cases, share examples of these with Datadog to inform future improvements. 

Reviewer: CTA? How should they share?  Slack etc?
